addons/resource.language.en_gb/resources/strings.po

```diff
@@ -16509,13 +16509,13 @@ msgstr ""
 #. Setting #38011 "Videos -> Library -> Show All Items entry"
 #: system/settings/settings.xml
 msgctxt "#38011"
-msgid "Show \"All Items\" entry"
+msgid "Show \"All items\" entry"
 msgstr ""
 
 #. Description of setting "Videos -> Library -> Show All Items entry"
 #: system/settings/settings.xml
 msgctxt "#38012"
-msgid "Show \"All Items\" entry in directory (for example All Albums or All Seasons)"
+msgid "Show \"All items\" entry in directory (for example All albums or All seasons)"
```

Comment:
while changing this, mind putting "All albums" and "All seasons" also in qoutes?